monai/handlers/mlflow_handler.py

```diff
@@ -100,6 +117,42 @@ def __init__(
         self.global_epoch_transform = global_epoch_transform
         self.state_attributes = state_attributes
         self.tag_name = tag_name
+        self.experiment_name = experiment_name
+        self.run_name = run_name
+        self.experiment_param = experiment_param
+        self.artifacts = ensure_tuple(artifacts)
+        self.optimizer_param_names = ensure_tuple(optimizer_param_names)
+        self.client = mlflow.MlflowClient()
+
+    def _is_param_exists(self, param_name: str) -> bool:
+        """
+        Check whether a parameter has already been logged in current mlflow run.
+
+        Args:
+            param_name: name of parameter to check in mlflow run.
+
+        Return:
+            If the given parameter already was logged to mlflow.
+        """
+        cur_run = mlflow.active_run()
+        log_data = self.client.get_run(cur_run.info.run_id).data
```

Comment:
this active_run/get_run are called for every key in key_list, perhaps merge`_is_param_exists` into `_delete_exist_param_in_dict`